config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -1340,6 +1340,133 @@
       "yBucketNumber": null,
       "yBucketSize": null
     },
+    {
+      "id": 31,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 90
+      },
+      "type": "heatmap",
+      "title": "Snapshot Created to PipelineRun With Ephemeral Environment Started Seconds",
+      "pluginVersion": "9.1.6",
+      "maxDataPoints": 25,
+      "description": "Time duration from the moment the snapshot resource was created till a integration pipelineRun is started for pipelineRuns with ephemeral environments",
+      "legend": {
+        "show": false
+      },
+      "cards": {
+        "cardPadding": null,
+        "cardRound": null
+      },
+      "color": {
+        "cardColor": "#b4ff00",
+        "colorScale": "sqrt",
+        "colorScheme": "interpolateOranges",
+        "exponent": 0.5,
+        "mode": "spectrum"
+      },
+      "dataFormat": "tsbuckets",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "hideFrom": {
+              "tooltip": false,
+              "viz": false,
+              "legend": false
+            }
+          }
+        },
+        "overrides": []
+      },
+      "heatmap": {},
+      "hideZeroBuckets": true,
+      "highlightCards": true,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(increase(snapshot_created_to_pipelinerun_with_ephemeral_env_started_seconds_bucket[$__interval])) by (le)",
+          "format": "heatmap",
+          "interval": "",
+          "legendFormat": "{{le}}",
+          "refId": "A",
+          "datasource": {
```

Comment:
Datasource must be removed from JSON

https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring#4-grafana-dashboards

Reply:
I don't see that in my local branch, I must have forgotten to push after removing it.  Should be all set now.